### ISSUE TITLE
Angular copy to clipboard icon focus color is incorrect

### ISSUE THREAD

**Issue by vesivagu** [angular/angular#37817]
https://angular.io/start#next-steps

![image](https://user-images.githubusercontent.com/25737946/85986066-4565e880-ba09-11ea-9580-4df4325cca0f.png)

Windows 10 
Google Chrome - Version 83.0.4103.116 (Official Build) (64-bit)

it should be blue color.

**Comment by gkalpak** [comment]
May I ask why it should be blue? 😕

**Comment by vesivagu** [comment]
i have been seeing -webkit-focus-ring-color should be by default #4d90fe (Google Chrome - Version 81.0)

but in latest chrome version this color has been changed to below, intentionally it has been changed?

![image](https://user-images.githubusercontent.com/25737946/86091307-4cecc680-bac9-11ea-9d2a-42efc9edd3b6.png)

**Comment by gkalpak** [comment]
So, if I understand correctly this is a style applied by the browser, not us. The question is whether we want to leave this default style or change it to something that better aligns with the overall design.

I'll defer to @sjtrimble :grin:

**Comment by vesivagu** [comment]
my opinion, for better ux we should use this color  #4d90fe for -webkit-focus-ring-color as Angular mostly uses blue related colors in the themes

**Comment by sjtrimble** [comment]
I agree with @vesivagu in that it would match our UI better and be more consistent if it were blue.

It should match our focus styles - it may remove outline and do a shadow possibly? Would have to check.

Should I update or do you want to @vesivagu?

**Comment by vesivagu** [comment]
@sjtrimble you can update
